routes/web.php

```diff
@@ -130,7 +130,6 @@
     Route::get('/', [AdminController::class, 'index']);
 
     // Users
-    Route::get('users/{username}', [UsersController::class, 'show'])->name('.users.show');
```

Comment:
This is a dead route detected by Enlightn. The `UsersController` doesn't have a `show` method any more. Perhaps, it used to exist.